paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/PaymentSheetScreen.kt

```diff
@@ -289,6 +290,8 @@ private fun PaymentSheetContent(
                 modifier = Modifier.padding(horizontal = horizontalPadding).padding(bottom = 8.dp),
             )
         }
+        
+        Spacer(modifier = Modifier.height(currentScreen.bottomContentPadding))
```

Comment:
As a result of the new screenshots, I found this issue where the error message was being moved up with the padding changes. Fixed with this PR.